prt_partner_tweaks_access/tests/common.py

```diff
@@ -0,0 +1,54 @@
+from odoo.tests.common import TransactionCase
+
+
+class PartnerTweaksAccessCommon(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super(PartnerTweaksAccessCommon, cls).setUpClass()
```

Comment:
Disable tracking unless you need it for functional purposes

    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))

Check the Dev Memo for details

Reply:
okay, I'll fix it